_search-plugins/sql/ppl/functions.md

```diff
@@ -771,3 +771,186 @@ sepal_length_in_cm | sepal_width_in_cm | petal_length_in_cm | petal_width_in_cm
 | 5.1 | 3.5 | 1.4 | 0.2 | 1   
 | 5.6 | 3.0 | 4.1 | 1.3 | 0
 | 6.7 | 2.5 | 5.8 | 1.8 | 2
+
+## join (experimental)
+
+Using `join` command to combines two datasets together. The left side could be an index or results from a piped commands, the right side could be either an index or a subquery.
+
+### Syntax
+
+```sql
```

Comment:
Why you specify the language as sql?

Reply:
if remove it, we have to remove all 56 ` ```sql ` in this file, and more in other files.

Reply:
I think we can keep these for now.